### PR TITLE
Feature/679 credit card payment receipts

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**
[App#679](https://app.zenhub.com/workspaces/fecfile-online-619e578e68408b001c831251/issues/gh/fecgov/fecfile-web-app/679)

[App PR](https://github.com/fecgov/fecfile-web-app/pull/889)
[API PR ](https://github.com/fecgov/fecfile-web-api/pull/360)
[Validator PR](https://github.com/fecgov/fecfile-validate/pull/186)

This branch is also used in connection with [another front-end branch](https://github.com/fecgov/fecfile-web-app/pull/893)

### CODE REVIEW

File: schema/DISBURSEMENT_PARENTS.json [hunk 1]
```diff
@@ -0,0 +1,538 @@
...
+            "const": "ORG",
+            "examples": [
+                "ORG"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "ENTITY TYPE",
+                "TYPE": "A/N-3",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "ORG",
+                "VALUE_REFERENCE": "ORG",
```
Comment (dheitzer) [inline schema/DISBURSEMENT_PARENTS.json:166]: in spec this looks like it should be [ORG]

File: schema/DISBURSEMENT_PARENTS.json [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+                "RULE_REFERENCE": "ORG",
```
Comment (dheitzer) [inline schema/DISBURSEMENT_PARENTS.json:167]: in spec this looks like it should be ORG only

File: schema/DISBURSEMENT_PARENTS.json [hunk 3]
```diff
@@ -0,0 +1,538 @@
...
+            ],
+            "examples": [
+                "Credit Card: See Below"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "EXPENDITURE PURPOSE DESCRIP",
+                "TYPE": "A/N-100",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "Credit Card: See Below",
+                "VALUE_REFERENCE": null,
```
Comment (dheitzer) [inline schema/DISBURSEMENT_PARENTS.json:387]: In spec this should be "Credit Card: See Below |
Payroll: See Below"

File: schema/DISBURSEMENT_PARENTS.json [hunk 4]
```diff
@@ -0,0 +1,538 @@
...
+            ],
+            "examples": [
+                "SB21b"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "FORM TYPE",
+                "TYPE": "A/N-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "SB21b",
+                "VALUE_REFERENCE": "SB[line# ref]",
```
Comment (dheitzer) [inline schema/DISBURSEMENT_PARENTS.json:41]: In spec is [SB21b | SB29]

Comment (Elaine-Krauss-TCG) [reply schema/DISBURSEMENT_PARENTS.json]: Thanks for the feedback; these issues have been addressed!

**Review by mjtravers — APPROVED** [review]

**Pull request merged**